ci-operator/infra/ansible-operator-imagestream.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```

Comment:
@stevekuznetsov is this file supposed to exist in ci-operator/infra or one of the sub-directories? I didn't see anything in the directory structure that was obvious to me.